lib/common/xml.c

```diff
@@ -1189,6 +1189,68 @@ crm_xml_set_id(xmlNode *xml, const char *format, ...)
     free(id);
 }
 
+/*!
+ * \internal
+ * \brief Write a string to a file stream, compressed using \c bzip2
+ *
+ * \param[in,out] text       String to write
+ * \param[in]     filename   Name of file being written (for logging only)
+ * \param[in,out] stream     Open file stream to write to
+ * \param[out]    bytes_out  Number of bytes written (valid only on success)
+ *
+ * \return Standard Pacemaker return code
+ *
+ * \note bzlib uses \p text as a <tt>void *</tt>, so there's no guarantee it
```

Comment:
This might not be worth calling it. It's no different from libxml2 and glib functions that take a non-const argument and have no reason to alter it.